learning-development/r.qmd

```diff
@@ -592,6 +592,21 @@ Then restart R-Studio and downloads with renv should now succeed.
 
 ---
 
+### Packages not downloading (or not able to connect to other external service from within R, Python, Bash etc.) due to SSL certificate error
```

Comment:
Is there a more concise title you could give this? Thinking the nav is already getting a bit out of hand as it is for section headers being on multiple lines
![image](https://github.com/dfe-analytical-services/analysts-guide/assets/52536248/b00edbae-10c5-4d65-a250-251b2bca2ba9)

Maybe something like 'SSL connection errors'?

Reply:
Fair point... I'll shorten it!